exercises/queen-attack/src/Queens.hs

```diff
@@ -0,0 +1,7 @@
+module Queens (boardString, canAttack) where
+
+boardString :: Maybe (Int, Int) -> Maybe (Int, Int) -> String
+boardString = undefined
```

Comment:
I didn't spend too much time writing the *stubs*, so most of them can probably be improved.
In this one, I just wrote the signature that was compatible with the tests.

From the signature, it's not possible to know that:

- The first is White and the second is Black, as you pointed out. 
- The indexes are (column, row)
- The board starts at (0,0), which is probably completely unnatural for a chess player.

> ... but maybe that's OK it's easily discoverable from reading the tests. unless the goal is to make it obvious without having to read the tests

I still don't know the answer to that. At first, I was just trying to save the users the repetitive work writing boilerplate code, but maybe that is a good idea...

Do you think we should change it to avoid the need to look at the tests?
If you write a PR, I will merge it.